include/nil/blueprint/assigner.hpp

```diff
@@ -1417,8 +1417,6 @@ namespace nil {
 
                             curr_branch.pop_back();
 
-                            ASSERT((stack_size - 1) == call_stack.size() || finished);
```

Comment:
I can see the same assert in this handler above. Is it still actual?

Reply:
good catch, fixed